src/plasma/plasma.cc

```diff
@@ -20,3 +20,13 @@ void warn_if_sigpipe(int status, int client_sock) {
   }
   LOG_FATAL("Failed to write message to client on fd %d.", client_sock);
 }
+
+uint8_t *create_object_info_buffer(ObjectInfoT *object_info) {
```

Comment:
Please document this saying that it mallocs and that it's the callers responsibility to free the returned buffer.